### DISCUSSION TITLE
Get list of ASINs through a list of SKUs

### DISCUSSION THREAD

**Discussion by paulamascot** [amzn/selling-partner-api-models#3409] [General] [answered]
Hi,

I need to get a list of ASINs from Amazon via API. I'm trying to use the Selling Partner API in order to get this list of ASINs from the API. I have a list of SKUs that I can use to find the ASINs. Can you help me and recommend which endpoint from Selling Partner API I should use to obtain this list of ASINs, please? Thank you!

**Comment by syedhsan** [comment]
@paulamascot 
Hey mate can you please tell me where you get SKUs because we have ASIN and we are in need of SKUs

**Comment by wszwc** [comment]
> @paulamascot 嘿伙计，你能告诉我你在哪里获得 SKU吗，因为我们有 ASIN 并且我们需要 SKU


How are you, brother? I have encountered the same problem. When I synchronize a single product, I get all the asin of the child variant through the parent asin. But if I want to get the data of the child variant, I need sku.

**Comment by chapmanjw** [comment, marked as answer]
@paulamascot If you are looking to retrieve ASIN data based on the SKUs you have for a seller, you can use the Catalog Item API `searchCatalogItems` operation. By using the `sellerId`, `identifiers`, and `identifiersType` parameters, you can provide a list of up-to 20 SKUs per request to find/retrieve the corresponding ASIN data: https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v2022-04-01-reference#identifierstype
